packages/lib-classifier/src/components/Classifier/components/SubjectViewer/components/VariableStarViewer/VariableStarViewer.js

```diff
@@ -108,20 +109,27 @@ const VariableStarViewer = React.forwardRef(function VariableStarViewer(props, r
       })}
       </Box>
       <Box
+        as='figure'
         direction='column'
         height='260px'
         gridArea='HRDiagram'
+        margin='none'
         width='220px'
       >
         <SingleImageViewer
-          height={260}
+          aria-labelledby='imageId'
+          height={230}
           enableInteractionLayer={false}
+          role='image'
```

Comment:
role=‘img’

Reply:
doh 🤦‍♀️

Reply:
This was my issue. It's reading out as expected now.